python-contact-book/source_code_final/README.md

```diff
@@ -0,0 +1,51 @@
+# RP Contacts
+
+**RP Contacts** is a Contact Book application built with Python, [PyQt5](https://www.riverbankcomputing.com/static/Docs/PyQt5/index.html), and [SQLite](https://www.sqlite.org/docs.html).
+
+## Running the Application
+
+To run **RP Contacts**, you need to download the sorce code. Then open a terminal or command-line window and run the following steps:
+
+1. Create and activate a Python virtual environment
+
+```sh
+$ cd rpcontacts/
+$ python3 -m venv ./venv
+$ source venv/bin/activate
+(venv) $
+```
+
+2. Install the dependencies
+
+```sh
+(venv) $ pip install -r requirements.txt
+```
+
+3. Run the application
+
+```sh
+(venv) $ python3 rpcontacts.py
```

Comment:
Since you're in the virtual environment here, you can just use the `python` command.

```suggestion
(venv) $ python rpcontacts.py
```